doc/SuppTechInfo/groundwater-energy-model.tex

```diff
@@ -0,0 +1,149 @@
+
+The dispersion term handled by the CND Package is given by the third expression listed in equation \ref{eqn:fterms}. The term in parentheses is the negative of the dispersive flux, which includes mechanical dispersion in the water and conduction in the water and solid. Accordingly, the dispersion tensor, $\matr{D}_T$, is expressed as the sum of contributions from mechanical dispersion and conduction:
+
+\begin{equation}
+\label{eqn:dt_tensor}
+\begin{aligned}
+\matr{D}_T = S_w \theta \matr{D}^{mech} + D^{mol} \matr{I},
+\end{aligned}
+\end{equation}
+
+\noindent where $\matr{D}^{mech}$ is the mechanical dispersion tensor,
+
+\begin{equation}
+\label{eqn:dmolforheat}
+\begin{aligned}
+D^{mol} = \frac {{k_T}_{bulk}}{\rho_w C_{pw}}
+\end{aligned}
+\end{equation}
+
+\noindent is a molecular diffusion coefficient ($L^2/T$) that combines the conductive capacity of the water and the solid, and
+
+\begin{equation}
+\label{eqn:ktbulk}
+\begin{aligned}
+{k_T}_{bulk} = S_w \theta {k_T}_w + \left ( 1 - \theta \right ) {k_T}_s
+\end{aligned}
+\end{equation}
+
+\noindent is a bulk thermal conductivity ($E/(T L \, deg)$) that depends on the thermal conductivities of the water and the solid, ${k_T}_w$ and ${k_T}_s$  ($E/(T L \, deg)$, respectively. The particular form of the bulk conductivity in equation \ref{eqn:ktbulk} corresponds to the \textit{ad hoc} assumption that the total conductive flux is the sum of independent contributions from the water and the solid, which are proportional to the volumes of water and solid, respectively. One could describe different conductive behavior using a nonlinear expression for bulk conductivity {\cite{campbell1994, markle2006}.  However, the bulk conductivity expression in equation \ref{eqn:ktbulk} is the only one available in the version of the GWE Model described here. Note that the CND Package does not account explicitly for tortuosity. The user should adjust the input values of ${k_T}_w$ and ${k_T}_s$ to account for the effects of tortuosity if desired.
+
+Once the dispersion tensor is calculated, the CND Package handles dispersion in a way that is directly analogous to the method used in the DSP Package. As in the DSP Package, the XT3D option is the default method for representing dispersion in the CND Package, and XT3D can be turned off by the user in package input.
+
+\subsubsection{Advanced Stress Packages}
+
+The version of the GWE Model described here offers four advanced stress packages, each of which is analogous and functions similarly to an advanced package in the GWT Model: Lake Energy Transport (LKE), Multi-Aquifer Well Energy Transport (MWE), Streamflow Energy Transport (SFE), and Unsaturated Zone Energy Transport (UZE). The GWE advanced stress packages will not be described in detail here except to note differences from the corresponding GWT packages. The primary difference, of course, is that GWE advanced stress packages deal with transport of thermal energy, not solute mass.
```

Comment:
Should mention that these packages explicitly simulate heat transport in their features; however, the SSM could be used instead to represent their effects on the subsurface if they have a known temperature.